addon/templates/components/cfb-form-editor/question-list.hbs

```diff
@@ -96,5 +93,8 @@
         </p>
       </li>
     {{/each}}
+    {{#if (and this.hasNextPage (eq this.mode "add"))}}
+      <li class="uk-text-center cfb-pointer"><a {{on "click" (perform this.data)}} href="#">{{t "caluma.form-builder.question.loadMore"}}</a></li>
```

Comment:
You'll need to prevent the default action of the event in your task

Reply:
Done